frontend/src/routes/[[lang=locale]]/(voters)/Header.svelte

```diff
@@ -0,0 +1,114 @@
+<script lang="ts">
+  import {t} from '$lib/i18n';
+  import Banner from './Banner.svelte';
+  import {Icon} from '$lib/components/icon';
+  import {AppLogo} from '$lib/templates/parts/appLogo';
+  import type {BasicPageProps} from '$lib/templates/basicPage/BasicPage.type';
+  import {Breakpoints} from '$lib/utils/breakpoints';
+  import {darkMode} from '$lib/utils/darkMode';
+  import {settings} from '$lib/stores';
+  import {onDestroy} from 'svelte';
+  import {getLayoutContext} from '$lib/contexts/layout';
+
+  export let navId: BasicPageProps['navId'] = 'pageNav';
+
+  export let openDrawer: () => void;
+  export let drawerOpen = false;
+  export let drawerOpenElement: HTMLButtonElement | undefined;
+
+  /** We use `videoHeight` and `videoWidth` as proxies to check for the presence of content in the `video` slot. Note that we cannot merely check if the slot is provided, because it might be empty. */
```

Comment:
I guess we should comment these video-related things out, as well as the invertLogo part below.